hardware-types/pcb/fileTypes.json

```diff
@@ -0,0 +1,4 @@
+{
+    "fileTypes" : ["gbr", "pdf", "drl", "txt", "csv"]

```

Comment:
@Bucknalla should we add xlsx, zip here as well?
Plus SchDoc, and other Altium ones?

Reply:
@anujdeshpande did we get a comprehensive list from the EtcherPro guys?

Reply:
I see the following in [EP's drive folder](https://drive.google.com/drive/u/0/folders/1Ps1dsA8fveKnyBnzIjDWWAXcwzFug-a7). I think it's safe to include atleast these and then see what breaks @Bucknalla 
- xlsx
- zip (3D and gerbers)
- SchDoc
- BomDoc
- PcbDoc
- PrjPcbStructure
- OutJob

Reply:
cc @zisis00 @ntzovanis

Reply:
@anujdeshpande not sure what you need here. Is it design files extensions?

Reply:
@ntzovanis yepp sorry for not giving more context - but this will help build RFQs for PCB projects

Reply:
@anujdeshpande Please add also the “.PrjPcb”. For the “.PrjPcbStructure” is not necessary as it is created when you open the “.PrjPcb”.